tensorflow/tsl/util/tensor_bundle/BUILD

```diff
@@ -22,10 +22,25 @@ cc_library(
     ],
 )
 
+filegroup(
+    name = "framework_internal_private_hdrs",
+    srcs = [
+        "byte_swap_array.h",
```

Comment:
You may as well as the `byte_swap_tensor.h` utility here too, for consistency (but again, moved under tsl/util).

Reply:
Sure, I'll move them to `tsl/util` folder as well. Thanks!

Reply:
okay, sounds good